runtime/lua/vim/lsp/util.lua

```diff
@@ -171,25 +171,24 @@ end
 ---@param encoding string utf-8|utf-16|utf-32| defaults to utf-16
 ---@return integer byte (utf-8) index of `encoding` index `index` in `line`
 function M._str_byteindex_enc(line, index, encoding)
```

Comment:
looks like we currently don't have a direct test of this , but worth adding to `test/functional/plugin/lsp/utils_spec.lua` ?

Reply:
#30735 will remove the need for this wrapper function